api/AudioParam.json

```diff
@@ -282,7 +282,9 @@
               "version_added": "14"
             },
             "chrome_android": {
-              "version_added": "18"
+              "version_added": "18",
+              "partial_implementation": true,
+              "notes": "This sets the target volume at the specified time, but it doesn’t ramp to it, causing this function to behave like <code>setValueAtTime()</code>."
```

Comment:
Still think this is wrong.  It is possible, however, for an implementation not to have implemented all the little bits.  The behavior of exponentialRampToValue has changed a bit over time where we've updated and/or clarified the behavior for exceptional cases.  An implementation may not have picked up those changes.